dapps/assets/src/components/Assets.vue

```diff
@@ -0,0 +1,88 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+<template>
+  <div class="evan theme-evan">
+    <evan-dapp-wrapper>
+      <template v-slot:content>
+        <evan-dapp-wrapper-level-2 ref="level2Wrapper">
+          <div class="sidenav">
+            <!-- Not using nav-list because it doesnt support router-link properly -->
+            <div class="evan-nav-list">
```

Comment:
Why not using `<evan-nav-list :entries="navEntries"></evan-nav-list>`? If you are unconformable with the setup of the links `href`, we should discuss to adjust the `evan-nav-list` component within the core to use the correct routing parameters.

Reply:
Since it requires some more refactoring and testing of backwards compatibility, I would add this as a todo for now. Can surely create a separate subtask for it.